tests/integration/test_storage_s3_queue/test.py

```diff
@@ -169,6 +170,25 @@ def started_cluster():
             image="clickhouse/clickhouse-server",
             tag="24.5",
             stay_alive=True,
+            main_configs=[
+                "configs/remote_servers_245.xml",
+            ],
+            user_configs=[
+                "configs/users.xml",
+            ],
+            with_installed_binary=True,
+            use_old_analyzer=True,
```

Comment:
Do we need to use old analyzer?

Reply:
idk, at one time this flag appeared here for all add_instance calls, added by someone I did not check who, and then I automatically reused it because was copy-pasting add_instance definition, will remove and check what fails.. may be in a separate pr